include/z64pause.h

```diff
@@ -26,6 +26,35 @@ typedef enum {
 #define PAUSE_EQUIP_BUFFER_SIZE sizeof(u16[PAUSE_EQUIP_PLAYER_HEIGHT][PAUSE_EQUIP_PLAYER_WIDTH])
 #define PAUSE_PLAYER_SEGMENT_GAMEPLAY_KEEP_BUFFER_SIZE 0x5000
 
+typedef enum {
+    /*  0 */ PAUSE_STATE_OFF,
+    /*  1 */ PAUSE_STATE_WAIT_LETTERBOX, // Request no letterboxing and wait for it.
+    /*  2 */ PAUSE_STATE_WAIT_BG_PRERENDER, // Wait for the pause background prerender to be done.
+    /*  3 */ PAUSE_STATE_INIT, // Load data and initialize/setup various things.
+    /*  4 */ PAUSE_STATE_OPENING_1, // Animate the pause menu coming together with rotations and other animations.
+    /*  5 */ PAUSE_STATE_OPENING_2, // Finish some animations for opening the menu.
+    /*  6 */ PAUSE_STATE_MAIN, // Pause menu ready for player inputs.
```

Comment:
maybe active instead of main? both work

Reply:
Something to consider is that there's a substate enum that will be needed for this state. Something like:
`PAUSE_MAIN_STATE_XXX`
or 
`PAUSE_ACTIVE_STATE_XXX`
which will be `pauseCtx->unk_1E4`

Reply:
Yeah this is where the scope can widen significantly, the substates of the main state (there also are substates for the save dialog state) involve logic from the equips page, which is a beast of its own. I could introduce dummy enums (named by number) though if it helps provide context

Reply:
I could also leave this state named as a number (`PAUSE_STATE_6`) until introducing the sub-state enum

Reply:
Well I did just that. sounds like the better idea

Reply:
For what is worth: in the LADX disassembly, when a system is finally ready after several stages of initialization, we named this kind of state "interactive". That would give something like `PAUSE_INTERACTIVE_STATE` in that case.

I find it descriptive enough. Of course it is lengthier to write (but I guess that could be abridged for sub-enums). And it doesn't perfectly convey that other states may also be responsive to user input, like the save prompt (but neither does "_ACTIVE").